### PR TITLE
Bug/#46 Fix for maven central publication

### PR DESCRIPTION
**Pull request by spyhunter99**
Ref #62 #27 

Note the new procedure is 
new procedure is
./gradlew clean
./gradlew install -Pprofile=javadoc,sources
./gradlew publishArtifacts -Pprofile=javadoc,sources,sign

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -297,10 +297,14 @@ That's a problem if you're trying to sign it. Unfortunately, we couldn't find a
...
 
+We're also not using the built in "publish" or "uploadArchive" task here because it does not 
+publish pom signatures correctly. It also uploads an additional variant of android libraries and APKs
+for no apparent reason and we can't find a way to sign those before uploading.
+
 ```bash
 $ ./gradlew clean
 $ ./gradlew install -Pprofile=sources,javadoc
-$ ./gradlew publish -Pprofile=sources,javadoc,sign
+$ ./gradlew publishArtifacts -Pprofile=sources,javadoc,sign
```
Comment (chrisdoyle) [inline README.md:307]: When in doubt, abandon ship and roll your own.  :)

File: gradle/maven-support.gradle [hunk 2]
```diff
@@ -1,12 +1,22 @@
...
  * Common Gradle script configuration for Maven tasks.
  *
  * @author Chris Doyle
  =================================================================================================*/
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+    dependencies {
+        classpath 'com.squareup.okhttp3:okhttp:3.4.1'
```
Comment (chrisdoyle) [inline gradle/maven-support.gradle:16]: Property-ize me?

File: gradle-fury-validation/build.gradle [hunk 3]
```diff
@@ -14,6 +14,7 @@ dependencies{
     compile group: 'commons-cli', name: 'commons-cli', version: '1.3'
 
 
+    compile group: 'org.jsoup', name: 'jsoup', version: '1.9.2'
```
Comment (chrisdoyle) [inline gradle-fury-validation/build.gradle]: Methinks this is a testCompile dependency.  Might want to also property-ize as well.

File: gradle/maven-support.gradle (hunk 2 above)
Comment (spyhunter99) [reply gradle/maven-support.gradle:16]: we can't, it's in the fury build script.

File: gradle-fury-validation/build.gradle (hunk 3 above)
Comment (spyhunter99) [reply gradle-fury-validation/build.gradle]: it isn't a test compile dependency, but can do on propertyize

File: gradle/maven-support.gradle (hunk 2 above)
Comment (chrisdoyle) [reply gradle/maven-support.gradle:16]: Haha, right!

**Pull request merged**